system/doc/reference_manual/typespec.md

```diff
@@ -352,7 +357,11 @@ module defining them is allowed to depend on their term structure. Consequently,
 such types do not make much sense as module local - module local types are not
 accessible by other modules anyway - and is always to be exported.
 
-Read more on [Opaques](opaques.md)
+Types declared as `nominal` are type-checked by Dialyzer according to the
+user-defined names. They are treated as regular types everywhere else, including
+the compiler.
```

Comment:
```suggestion
Types declared as `nominal` are type-checked according to the user-defined names instead of their structure. That is `-nominal feet() :: integer()` and `-nominal meter() :: integer()` are not the same type, while if `-type` is used it would be.
```

I think we should refrain from describing how tools interpret the attributes as much as possible, and instead describe what they mean in a tool-independent fasion. Above is an attempt at a rewrite :)